<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWindow</class>
 <widget class="QMainWindow" name="QtWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1463</width>
    <height>866</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Traveling Salesman</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1161</width>
      <height>801</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="grid"/>
   </widget>
   <widget class="QPushButton" name="btnCreate">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>10</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRedraw">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>130</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Перерисовать!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>250</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelete">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>490</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAnswer">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>610</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Маршрут</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEdit">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>370</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить!</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1463</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
